src/screens/Authentication/phonenumberinput.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useRef } from 'react';
```

Comment:
this should be camel case and should live in `components` rather than `screens/Authentication`